<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailPage</class>
 <widget class="QWidget" name="EmailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>593</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Email&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line16">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Google Mail</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="googleMailCheckBox">
        <property name="text">
         <string>Use Google Mail with OAuth2 authentication</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="googleMailInfoLabel">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; Google Mail using SMTP should be configured below.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Standard Email</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel2_2">
          <property name="text">
           <string>Outgoing mail server (SMTP):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="mailServerEdit">
          <property name="toolTip">
           <string>Enter the address of your mail server</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Encryption Method:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="noEncryptionButton">
             <property name="toolTip">
              <string>Select to use no encryption</string>
             </property>
             <property name="text">
              <string>None</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="useSslButton">
             <property name="toolTip">
              <string>Select to use SSL</string>
             </property>
             <property name="text">
              <string>SSL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="useTlsButton">
             <property name="toolTip">
              <string>Select to use TLS</string>
             </property>
             <property name="text">
              <string>TLS</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="portSpin">
          <property name="toolTip">
           <string>Enter the port of the mail server</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="textLabel1_5">
          <property name="text">
           <string>Email address:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="emailEdit">
          <property name="toolTip">
           <string>Enter your email address</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="textLabel1_6">
          <property name="text">
           <string>Signature:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QTextEdit" name="signatureEdit">
          <property name="toolTip">
           <string>Enter your email signature</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="mailAuthenticationGroup">
        <property name="toolTip">
         <string>Select to authenticatate against the mail server</string>
        </property>
        <property name="title">
         <string>Mail server needs authentication</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="textLabel1_15">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mailUserEdit">
           <property name="toolTip">
            <string>Enter your mail server username</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel2_7">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mailPasswordEdit">
           <property name="toolTip">
            <string>Enter your password for accessing the mail server</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="testButton">
           <property name="toolTip">
            <string>Press to test the login data</string>
           </property>
           <property name="text">
            <string>Test Login</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>googleMailCheckBox</tabstop>
  <tabstop>mailServerEdit</tabstop>
  <tabstop>noEncryptionButton</tabstop>
  <tabstop>useSslButton</tabstop>
  <tabstop>useTlsButton</tabstop>
  <tabstop>portSpin</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>signatureEdit</tabstop>
  <tabstop>mailAuthenticationGroup</tabstop>
  <tabstop>mailUserEdit</tabstop>
  <tabstop>mailPasswordEdit</tabstop>
  <tabstop>testButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>googleMailCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
